frontend/amundsen_application/static/js/components/Alert/AlertList.tsx

```diff
@@ -10,19 +10,48 @@ export interface AlertListProps {
   notices: NoticeType[];
 }
 
+export interface AggregatedAlertListProps {
+  notices: {
+    [key: string]: NoticeType;
+  };
+}
+
+const aggregateNotices = (notices) =>
+  notices.reduce((accum, notice: any) => {
+    if (notice) {
+      const { messageHtml, severity, payload } = notice;
+
+      /* eslint-disable no-param-reassign */
```

Comment:
We could get rid of this rule, we have better things to care about!

Reply:
Sounds good!

Reply:
Adjusted the rule to turn it off